notebooks/experimental/prediction_tensorflow_model.ipynb

```diff
@@ -8,6 +8,14 @@
     "# Unfolded SDK Machine Learning Demo"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "05c04f6f-6bdb-4cf6-8a4e-64639c262a74",
+   "metadata": {},
+   "source": [
+    "The purpose of the model is to predict the median_house_value in California and then clustering it by the predicted prices."
```

Comment:
How about "This example demonstrates how Unfolded Map SDK can be used in the process of building a machine learning model for predicting median house prices in California."?

Reply:
good idea, thanks!